use-cases.md

```diff
@@ -125,3 +125,24 @@ Mobile Key solutions require a state of reservation in real time, a key should n
 
 After receiving a websocket event, use [Get all reservations by ids](operations/reservations.md#get-all-reservations-by-ids) to retrieve information about the reservation and customer if the websocket event fits your criteria. With this response, you will have information to issue the mobile key to the customer using their contact information in Mews.
 
+## Event Management Software
+
+An event management integration pushes contracted revenue to the relevant client’s profile in Mews, pulls live information about rates and availability and allows for the group bookings already pushed into Mews to be managed from the event management software. The sections below contain guidelines regarding the flow of information and the relevant endpoints used.
+
+### New/Existing Guest
+
+In Mews, billing is managed at the guest’s profile level instead of being placed on a specific room. In order to be able to send the charges to the correct guest, they must have a profile in Mews. A new guest can be added using the [Add customer](operations/customers.md#add-customer) operation and any information can be updated using the [Update customer](operations/customers.md#update-customer) one. In order to retrieve a list of all customers created within a certain interval, use the [Get all customers](operations/customers.md#get-all-customers) operation.
+
+### New Group Booking 
+
+When a new group booking is created within the event management software, this new reservation needs to be synced with Mews. This information can be pushed into Mews using the [Add reservations](operations/reservations.md#add-reservations) operation. 
+In order to ensure that the client can further manage individual companions to the group booking via the integration, use the [Add reservation companion](operations/reservations.md#add-reservation-companion) or [Delete reservation companion](operations/reservations.md#delete-reservation-companion). 
+
+### Adding Products to the Booking
+
+One of the expected functionalities of an event management integration is to push product and service orders into Mews to the correct customer’s profile. This can be done using the operation [Add order](operations/services.md#add-order). 
+
+### Rates and Availability 
+
+In order to pull live data about rates from Mews into the event management integration you can use the [Get all rates](operations/services.md#get-all-rates) and  [Get rate pricing](operations/services.md#get-all-rates) as well as [Get service availability](operations/services.md#get-service-availability).
```

Comment:
Here and above, you are talking about live data? This term is not used anywhere else. Why not just get/obtain/pull?